core/src/apps/cardano/address.py

```diff
@@ -74,7 +83,7 @@ def get_pointer_address(
     return get_human_readable_address(address)
 
 
-def get_enterprise_address(keychain, path: list, network_id):
+def get_enterprise_address(keychain: seed.Keychain, path: list, network_id: int) -> str:
```

Comment:
Do we know what are the elements of the list? Is it, e.g. List[int]? I would be explicit about element type wherever we use it.

Also, sometimes it is better to `Iterable` instead of `List`, which also covers Tuples and output from generators (yield).

Reply:
I sort of agree. However, I chose not to add the element type because it's mostly not used in the project (except protobuf types). Also this doesn't require any imports from `typing`.

We mostly know, what the types should be though, so I can add them if you think it's better. I am not yet bold/brave enough to do something differently :wink:

Reply:
Ok. Let's discuss this separately (whether to use typing or not).